specification/columns/skumeter.md

```diff
@@ -0,0 +1,33 @@
+# SKU Meter
+
+The SKU Meter describes the capability being metered or measured by a particular SKU in a charge.
+
+Providers often have billing models in which multiple SKUs exist for a given service to describe and bill for different capabilities for that service. For example, an object storage service may have separate SKUs for capabilities such as object storage, API requests, data transfer, encryption, object management. This field helps practitioners understand which capabilities are being metered by SKUs that appear in a FOCUS dataset.
+
+The SkuMeter column MUST be present in the billing data when a provider supports describing the capabilities contained within a SKU. The column MUST be of type String and MAY be null.
+
+## Column ID
+
+SkuMeter
+
+## Display Name
+
+SkuMeter
```

Comment:
```suggestion
SKU Meter
```

Reply:
fixed and updated to SKU Meter Name, thanks!